akka-remote-tests/src/multi-jvm/scala/akka/remote/RemoteNodeShutdownAndComesBackSpec.scala

```diff
@@ -118,8 +118,8 @@ abstract class RemoteNodeShutdownAndComesBackSpec
         watch(subjectNew)
 
         subjectNew ! "shutdown"
-        fishForMessage(5.seconds) {
-          case _: ActorIdentity       ⇒ false
+        receiveWhile(5.seconds) {
+          case _: ActorIdentity       ⇒ true
           case Terminated(subjectNew) ⇒ true
```

Comment:
That was my first idea, but then I thought you corrected me to do it like this, good because I didn't really understand how this would work (except for always waiting the 5 seconds).